Casks/h/helio.rb

```diff
@@ -0,0 +1,29 @@
+cask "helio" do
+  version "3.14"
+  sha256 "7d850e8d84b0723d3e253abb8e01b02ad3909f10209793c5169a60dd47a0ece4"
+
+  url "https://ci.helio.fm/helio-#{version}.dmg"
+  name "Helio"
+  desc "Music composition software"
+  homepage "https://helio.fm/"
+
+  livecheck do
+    url :homepage
+    regex(/helio[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

Comment:
```suggestion
    regex(/href=.*?helio[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

We usually restrict matching to `href` attributes (see [livecheck documentation](https://docs.brew.sh/Brew-Livecheck#file-names)).